### PR TITLE
Update Set-SPOTenant.md

### PR DESCRIPTION
**Pull request by msjennywu**
1. Fix typos
3. Fix MajorVersionLimit value range

### CODE REVIEW

**Comment by msjennywu** [comment]
Hi @JohanFreelancer9 @dariomws @yogkumgit @tiburd , Could you please review this change when you get a chance? Thanks!

**Review by dariomws — APPROVED** [review]
@msjennywu, I approve but couldn't test it in my demo tenant as EnableAutoExpirationVersionTrim is not available.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -2465,10 +2465,10 @@ The valid values are:
...
 
 > [!NOTE]
-> When Version History Limits are managed Manually (EnableAutoExpirationVersionTrim $false), MajorVersionLimits and ExpireVersionsAfterDays are both required parameters with the following acceptable values:<br/>
-> a. MajorVersionLimits that will accept values between 100 and 50,000 and<br/>
-> b. ExpireVersionsAfterDays will accept values of 0 to Never Expire or values >= 30 to delete versions that exceed that time period.<br/><br/>
-> When Version History Limits are managed Automatically (EnableAutoExpirationVersionTrim $true), setting MajorVersionLimits or ExpireVersionsAfterDays will result in an error as the count limits are set by the service.
+> When Version History Limits are managed Manually (EnableAutoExpirationVersionTrim $false), MajorVersionLimit and ExpireVersionsAfterDays are both required parameters with the following acceptable values:<br/>
+> a. MajorVersionLimit accepts values from 1 throuth 50,000 (inclusive).<br/>
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2469]:
```suggestion
> a. MajorVersionLimit accepts values from 1 through 50,000 (inclusive).<br/>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msjennywu, one more edit to fix the typo, please.

**Comment by tiburd** [comment]
> Apologies @msjennywu, one more edit to fix the typo, please.

I reviewed and accepted it.

**Pull request merged**